src/cpu.c

```diff
@@ -1,497 +1,633 @@
+    LD_REG_REG(&c.B, c.D);
+}
+
+void OPC_LD_B_E(void) {
+    LD_REG_REG(&c.B, c.E);
+}
+
+void OPC_LD_B_H(void) {
+    LD_REG_REG(&c.B, c.H);
+}
+
+void OPC_LD_B_L(void) { 
+    LD_REG_REG(&c.B, c.L);
+}
+
+void OPC_LD_B_HL(void) { 
+    LD_REG_REG(&c.B, mmu_get_byte(get_HL()));
+    c.cycle_count++;
+}
+
+void OPC_LD_B_A(void) {
+    LD_REG_REG(&c.B, c.A);
+}
+
+void OPC_LD_C_B(void) { 
+    LD_REG_REG(&c.C, c.B);
+}
+
+void OPC_LD_C_C(void) {
+    LD_REG_REG(&c.C, c.C);
+}
+
+void OPC_LD_C_D(void) { 
+    LD_REG_REG(&c.C, c.D);
+}
+
+void OPC_LD_C_E(void) {
+    LD_REG_REG(&c.C, c.E);
+}
+
+void OPC_LD_C_H(void) { 
+    LD_REG_REG(&c.C, c.H);
+}
+
+void OPC_LD_C_L(void) { 
+    LD_REG_REG(&c.C, c.L);
+}
+
+void OPC_LD_C_HL(void) { 
+    LD_REG_REG(&c.C, mmu_get_byte(get_HL()));
+    c.cycle_count++;
+}
+
+void OPC_LD_C_A(void) { 
+    LD_REG_REG(&c.C, c.A);
+}
+
+void OPC_LD_D_B(void) { 
+    LD_REG_REG(&c.D, c.B);
+}
+
+void OPC_LD_D_C(void) { 
+    LD_REG_REG(&c.D, c.C);
+}
+
+void OPC_LD_D_D(void) { 
+    LD_REG_REG(&c.D, c.D);
+}
+
+void OPC_LD_D_E(void) { 
+    LD_REG_REG(&c.D, c.E);
+}
+
+void OPC_LD_D_H(void) { 
+    LD_REG_REG(&c.D, c.H);
+}
+
+void OPC_LD_D_L(void) { 
+    LD_REG_REG(&c.D, c.L);
+}
+
+void OPC_LD_D_HL(void) { 
+    LD_REG_REG(&c.D, mmu_get_byte(get_HL()));
+    c.cycle_count++;
+}
+
+void OPC_LD_D_A(void) { 
+    LD_REG_REG(&c.D, c.A);
+}
+
+void OPC_LD_E_B(void) { 
+    LD_REG_REG(&c.E, c.B);
+}
+
+void OPC_LD_E_C(void) { 
+    LD_REG_REG(&c.E, c.C);
+}
+
+void OPC_LD_E_D(void) { 
+    LD_REG_REG(&c.E, c.D);
+}
+
+void OPC_LD_E_E(void) { 
+    LD_REG_REG(&c.E, c.E);
+}
+
+void OPC_LD_E_H(void) { 
+    LD_REG_REG(&c.E, c.H);
+}
+
+void OPC_LD_E_L(void) { 
+    LD_REG_REG(&c.E, c.L);
+}
+
+void OPC_LD_E_HL(void) { 
+    LD_REG_REG(&c.E, mmu_get_byte(get_HL()));
+    c.cycle_count++;
+}
+
+void OPC_LD_E_A(void) { 
+    LD_REG_REG(&c.E, c.A);
+    c.cycle_count++;
+}
+
+void OPC_LD_H_B(void) { 
+    LD_REG_REG(&c.H, c.B);
+}
+
+void OPC_LD_H_C(void) { 
+    LD_REG_REG(&c.H, c.C);
+}
+
+void OPC_LD_H_D(void) { 
+    LD_REG_REG(&c.H, c.D);
+}
+
+void OPC_LD_H_E(void) { 
+    LD_REG_REG(&c.H, c.E);
+}
+
+void OPC_LD_H_H(void) { 
+    LD_REG_REG(&c.H, c.H);
+}
+
+void OPC_LD_H_L(void) { 
+    LD_REG_REG(&c.H, c.L);
+}
+
+void OPC_LD_H_HL(void) { 
+    LD_REG_REG(&c.H, mmu_get_byte(get_HL()));
+    c.cycle_count++;
+}
+
+void OPC_LD_H_A(void) { 
+    LD_REG_REG(&c.H, c.A);
+}
+
+void OPC_LD_L_B(void) { 
+    LD_REG_REG(&c.L, c.B);
+}
+
+void OPC_LD_L_C(void) { 
+    LD_REG_REG(&c.L, c.C);
+}
+
+void OPC_LD_L_D(void) { 
+    LD_REG_REG(&c.L, c.D);
+}
+
+void OPC_LD_L_E(void) { 
+    LD_REG_REG(&c.L, c.E);
+}
+
+void OPC_LD_L_H(void) { 
+    LD_REG_REG(&c.L, c.H);
+}
+
+void OPC_LD_L_L(void) { 
+    LD_REG_REG(&c.L, c.L);
+}
+
+void OPC_LD_L_HL(void) { 
+    LD_REG_REG(&c.L, mmu_get_byte(get_HL()));
+    c.cycle_count++;
+}
+
+void OPC_LD_L_A(void) { 
+    LD_REG_REG(&c.L, c.A);
+}
+
+void OPC_LD_HL_B(void) { 
+
```

Comment:
Maybe add something here that breaks execution and notifies you if this unimplemented function is used instead of an empty function could help.

Reply:
I'm aware that this is not the aim of this MR, this just is a general comment.

Reply:
I'll just remove the empty functions, therefore the initialization function triggers!
Therefore, the information gets printed